### ISSUE TITLE
Create Offline Report on Image Groups

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#2133]
Create a report similar to the new one on Batches and add to the new "Offline Reports" Action.

**Comment by Captainkirkdawson** [comment]
Presume you mean image group list

**Comment by edickens** [comment]
Yes.  Not images, but include levels above Image Groups.  With total number of images in each group and years, if entered.  TY E
